rclcpp_lifecycle/src/lifecycle_node_interface_impl.hpp

```diff
@@ -359,9 +359,9 @@ class LifecycleNode::LifecycleNodeInterfaceImpl
       } catch (const std::exception &) {
         // TODO(karsten1987): Windows CI doens't let me print the msg here
         // the todo is to forward the exception to the on_error callback
-        // fprintf(stderr, "Caught exception in callback for transition %d\n",
-        //  it->first);
-        // fprintf(stderr, "Original error msg: %s\n", e.what());
+        // RCUTILS_LOG_ERROR("Caught exception in callback for transition %d\n",
```

Comment:
I'd say this should use _NAMED for consistency with the other logging calls

Reply:
All logging calls in this file are not using _NAMED so I matched it